crypto/crypto_test.ts

```diff
@@ -260,6 +266,35 @@ Deno.test("digest() keeps memory usage reasonable with many calls", async () =>
   );
 });
 
+Deno.test("digest() throws on invalid input", async () => {
+  const inputString = "taking the hobbits to isengard";
+  const inputBytes = new TextEncoder().encode(inputString);
+
+  await assertRejects(
+    () => stdCrypto.subtle.digest("BLAKE2B", {} as Iterable<never>),
+    TypeError,
+    "data must be a BufferSource or [Async]Iterable<BufferSource>",
+  );
```

Comment:
Can you please add the same test for `stdCrypto.subtle.digestSync()`? See https://app.codecov.io/gh/denoland/deno_std/pull/4620/blob/crypto/crypto.ts#L290

Reply:
ah, yes. sorry about that. added cases for `stdCrypto.subtle.digestSync()`.